include/mbedtls/pk.h

```diff
@@ -776,6 +776,53 @@ int mbedtls_pk_parse_public_keyfile( mbedtls_pk_context *ctx, const char *path )
 #endif /* MBEDTLS_PK_PARSE_C */
 
 #if defined(MBEDTLS_PK_WRITE_C)
+
```

Comment:
Seems like moving the macros to header file won't be so simple. CI is unhappy because of the macro names do not match pattern:

```
    > include/mbedtls/pk.h:796: 'RSA_PUB_DER_MAX_BYTES' does not match the
      required pattern '^(MBEDTLS|PSA)_[0-9A-Z_]*[0-9A-Z]$'
```

It would require to change the names of macros to `MBEDTLS_xxx` and adapt the existing code.

Reply:
Will be done in #5581 instead of currently pushed solution, will need to rebase when merged